examples/bpk-animate-height/stories.js

```diff
@@ -17,10 +17,13 @@
  */
 
 /* @flow strict */
```

Comment:
Should we remove the flow here?

Reply:
Hmm not sure why we have flow in the stories files tbh, we do still use flow, but not in the stories 😅 
I guess we could remove them since we're already making a change to every single story file 🤔

Reply:
done!